assets/js/18f-guide.js

```diff
@@ -0,0 +1,11 @@
+$(function() {
+  $('.sidebar-nav').each(function() {
+    accordion($(this));
+  });
+});
+
+// http://bryanbraun.github.io/anchorjs/
+anchors.options = {
+  visible: 'touch'
+};
+anchors.add('h3,h4,h5');
```

Comment:
This didn't do much in 18F/guides-template until I added `h2` to the mix, but started looking pretty _sweeet_ when I did. Any reason not to add it?